README.md

```diff
@@ -230,11 +243,11 @@ For a step-by-step guide on deploying production server with static files,
 
 ## Installation
 
-1. Install the app into your environment:
+1. Install `django_components` into your environment:
```

Comment:
Changed the wording here and there, so the purpose of each step is clearer. Also, what was previously step 5 (`STATICFILES_DIRS`) is now split into steps 4 and 6 (`COMPONENTS.dirs` and `STATICFILES_FINDERS`)